packages/shared/lib/src/sdk/wallet.rs

```diff
@@ -105,21 +37,15 @@ pub fn add_key(
 ///
 /// * `wallet` - Instance of a wallet struct.
 /// * `xsk` - String representing serialized ExtendedSpendingKey.
-/// * `password` - Password to encrypt the spending key.
 /// * `alias` - Spending key alias.
-pub fn add_spending_key(
-    wallet: &mut Wallet<BrowserWalletUtils>,
-    xsk: &str,
-    password: Option<String>,
-    alias: &str,
-) {
+pub fn add_spending_key(wallet: &mut Wallet<BrowserWalletUtils>, xsk: &str, alias: &str) {
```

Comment:
Do we still need this?

Reply:
Currently, yes, as I am still invoking it for transfers from a shielded account. I'm not sure to remove this, but I'm taking a look along with the shielded tx error in another issue